subsystem/exp_results/run.py

```diff
@@ -31,14 +32,30 @@ def failed_experiment_field(exp, stage_statuses, stage, duration=None, notify=No
 
     return build_field(title=exp, value=message)
 
+def send_graphs(config, info, client, output_dir):
+    os.chdir(output_dir)
```

Comment:
Is it necessary to change the directory if you already have the graph directory?